package.json

```diff
@@ -122,6 +122,7 @@
     "cross-env": "^7.0.2",
     "electron-installer-common": "^0.10.2",
     "eslint": "^8.0.1",
+    "eslint-plugin-ava": "^12.0.0",
```

Comment:
I think this is an error with `@malept/eslint-config` because we don't use Ava as a test runner for Forge.

Reply:
In `@malept/eslint-config` so the `eslint-plugin-ava` needs to became optional peer dependency; I will open an issue.